examples/api/test/material/selection_container/selection_container.0_test.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_api_samples/material/selection_container/selection_container.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('The SelectionContainer should transform the partial selection into an all selection', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.SelectionContainerExampleApp(),
+    );
+
+    expect(find.widgetWithText(AppBar, 'SelectionContainer Sample'), findsOne);
+
```

Comment:
Consider removing this extra line

Reply:
Sorry about that, I've done it in [format: Remove extra empty line](https://github.com/flutter/flutter/pull/157079/commits/49c731631c74dc673662229bdc2b2fd0670fbf54)